server/Arcadia.Assistant.InboxEmail.Abstractions/Email.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Arcadia.Assistant.InboxEmail.Abstractions
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    public class Email
+    {
+        public Email(uint uniqueId, string text, IEnumerable<Stream> attachments)
+        {
+            this.UniqueId = uniqueId;
+            this.Text = text;
+            this.Attachments = attachments;
+        }
+
+        public uint UniqueId { get; }
+
+        public string Text { get; }
+
+        public IEnumerable<Stream> Attachments { get; }
```

Comment:
is it a message? Stream is not something serializable, it's hard to manage its lifecycle. byte arrays?

Reply:
especially since its MemStream :D

Reply:
Fixed.